apps/docs/pages/guides/ai/hugging-face.mdx

```diff
@@ -59,27 +61,23 @@ To ensure you don't experience any unexpected downtime or errors, we recommend c
 
 </Admonition>
 
-## Integrate with Supabase
-
-Supabase aims to make it as easy as possible to integrate with third-party tools like Hugging Face.
-
-### Edge functions
+## Edge Functions
 
-When building any application, you should never trust the client frontend with secrets or credentials. Here this means you should never use your Hugging Face access token directly from your frontend. Supabase solves this using [Edge Functions](/docs/guides/functions), server side TypeScript functions that spin up on-demand only as you need them. Since Edge Functions run on a server, you can safely give them access to your Hugging Face access token.
+Edge Functions are server side TypeScript functions that spin up on-demand only as you need them. Since Edge Functions run on a server, you can safely give them access to your Hugging Face access token.
```

Comment:
```suggestion
Edge Functions are server-side TypeScript functions that run on-demand. Since Edge Functions run on a server, you can safely give them access to your Hugging Face access token.
```